examples/utils/example-utils/package.json

```diff
@@ -12,17 +12,80 @@
 	"license": "MIT",
 	"author": "Microsoft and contributors",
 	"sideEffects": false,
+	"type": "module",
+	"exports": {
+		".": {
```

Comment:
I suspect this guy doesn't have all of those. I think it is nice to just put out the ones that are relevant.

Reply:
@jason-ha - I'm going to gently push back on this, as this requires the developer introducing a new api tag to notice and fix the missing export.

Arguably, developers are responsible for reviewing the generated *.mds contain the expected changes, but as you've probably figured out from working with me these past couple weeks, not everyone is as sharp as you are at spotting inconsistencies in package.json and generated files.

Reply:
I think there is customer good to *not* have the extra exports to pick from.
`./alpha` and `./fruit` are a special case that we want to go away; so, I think we mostly exclude that one from making our choice about what is right. (no one should use it unless they know why they need to.)
Similarly, I expect `./internal` to exist a lot. That one is also not for customers and we should certainly notice if it isn't present.
`./public` is also a special case and temporary. (Right?) I think actually checking in any code that actually uses one of those will cause trouble. I am not sure we are really benefitting from having these temporarily. I have a related question from something I was working on yesterday - need to post after I get time to check on two more things.

`./beta` is the real thing we should probably base decision on. I posit that anyone adding @beta APIs will need to be quite aware of the pattern and should add tests that confirm working as expected. I am also in favor of sharpening our co-workers. ;)

Reply:
Down to 3 exports, which will become two when './public' becomes './'.